src/map/clif.h

```diff
@@ -660,6 +660,10 @@ enum stylist_shop {
 	STYLIST_SHOP_FAILURE
 };
 
+enum memorial_dungeon_command {
```

Comment:
seems to miss some entries in the enum 
**COMMAND_SUBSCRIPTION_CANCEL** and **COMMAND_MEMORIALDUNGEON_DESTROY**

Aegis also uses an enum Inter to zone

```c
enum PACKET_IZ_MEMORIALDUNGEON_COMMAND::<unnamed-tag>{
	COMMAND_PLAYER_KICK = 0x0
};
```

Reply:
the enum leaked from old aegis is incompatible with these values, to avoid headache currently i'm adding only this entry